### PR TITLE
dashboard: Updating env always uses latest release

### PR DESCRIPTION
**Pull request by jvatic**
#1389

### CODE REVIEW

**Comment by titanous** [comment]
LGTM

**Comment by jvatic** [comment]
Updated to copy SLUG_URL env from latest release.

File: dashboard/app/lib/javascripts/dashboard/stores/app.js [hunk 1]
```diff
@@ -66,8 +66,17 @@ var App = Dashboard.Stores.App = Dashboard.Store.createClass({
...
 		switch (event.name) {
 			case "APP_ENV:CREATE_RELEASE":
-				this.__createRelease(event.release).then(function () {
-					return this.__fetchAppRelease().then(this.__fetchAppFormation.bind(this));
+				this.__withoutChangeEvents(function () {
+					return this.__fetchAppRelease();
+				}.bind(this)).then(function () {
+					var release = Marbles.Utils.extend({}, this.state.release, event.release);
+					if (release.env.SLUG_URL) {
+						release.env.SLUG_URL = this.state.release.env.SLUG_URL;
```
Comment (titanous) [inline dashboard/app/lib/javascripts/dashboard/stores/app.js]: Please add a TODO about using CAS here. This is a hack.

**Comment by titanous** [comment]
Neat, LGTM!

**Pull request merged**